cves/kernel/CVE-2021-4032.yml

```diff
@@ -55,7 +55,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A key linuk feature is the KVM, the Kernel Virtual Machine, which allows you to create
+  an entire virtual machine including a virtual CPU. Included in the VCPU is a LAPIC otherwise 
+  know as a Local Advanced Interrupt Controller. An interupt controller is a program that allows 
+  interupt requests to be sent to the VCPU which allow the VCPU to interupt whatever current program
+  it is running to run a different program, including the construction of multiprocessor systems.
+
+  The vulnerbility is introduced when attempting to free the memory allocated for the LAPIC and the VCPU.
+  The VCPU sometimes will fail to correctly handle memory errors when being constructed, so when the program 
+  attempts to free the LAPIC wasn't created due to error, there is a system failure and it can crash the kernal, leading
```

Comment:
Wording gets a bit confusing in "so when the program...created due to error".

Reply:
I changed up the wording a bit so it's more readable, thank you for the feedback!